canonical-web-and-design/canonical.com
# 1593 build careers job application page

## Done
- Add careers job application page

## QA
- Run the website locally using `./run`
- Go to [/careers/apply](http://0.0.0.0:8002/careers/apply)
- Check if the `Locate me` button works
- Check if the education selectors work
- Check if you can add/remove education

## Issue

https://github.com/canonical-web-and-design/web-squad/issues/1593
